ci/playbooks/collect-logs.yml

```diff
@@ -53,6 +53,29 @@
                 cmd: |
                   ls -lRZ --hide=venv --hide=repo-setup {{ ansible_user_dir }}/ci-framework-data > ./selinux-listing.log;
 
+            - name: Set fact about kubelet systemctl use
+              become: true
+              ignore_errors: true # noqa: ignore-errors
+              delegate_to: crc
+              ansible.builtin.shell:
+                cmd: systemctl is-active kubelet
+              register: _collect_logs_systemctl_kubelet
+
+            - name: Get kubelet journalctl logs
+              ignore_errors: true  # noqa: ignore-errors
+              become: true
+              delegate_to: crc
+              ansible.builtin.shell:
+                cmd: journalctl -u kubelet;
+              register: _collect_logs_kubelet_journal_content
+
+            - name: Save crc kubelet journal
+              ansible.builtin.copy:
+                content: "{{ _collect_logs_kubelet_journal_content.stdout | trim }}"
+                dest: "{{ ansible_user_dir }}/zuul-output/logs/crc-kubelet.log"
+              delegate_to: localhost
```

Comment:
I might be mistaken, but the default `delegate_to` is used from the context of the current host (in that case it would be a `localhost`). Am I missing something here?